vscode/package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "qsharp-vscode",
-  "displayName": "Q# (new)",
-  "description": "Q# Language Support",
-  "version": "0.0.1",
+  "name": "qs",
+  "displayName": "MS/QS",
+  "description": "QS Lang Supp",
```

Comment:
The 3 lines above this don't see like names or descriptions we'd want to use.

Reply:
These were drafts. I will recover the original lines. 
Feel free to propose new values.